Makefile

```diff
@@ -85,6 +85,20 @@ build-esp32-bin:
 		--merge \
 		target/xtensa-esp32-espidf/release/micro-rdk-server-esp32.bin
 
+build-esp32c6-bin:
```

Comment:
I don't love how this Makefile is evolving but I don't have a better answer just yet. Thinking about it.